server/src/middleware/http/proxy_get_request.rs

```diff
@@ -149,8 +150,12 @@ where
 			(Some(method), &Method::GET) => {
 				// RPC methods are accessed with `POST`.
 				*req.method_mut() = Method::POST;
-				// Precautionary remove the URI.
-				*req.uri_mut() = Uri::from_static("/");
+				// Precautionary remove the URI path.
```

Comment:
//cc @lexnv do you remember why we removed the URI in the first place?

Reply:
I don't quite remember 🤔